web-admin/src/routes/-/embed/+layout.svelte

```diff
@@ -54,10 +65,26 @@
     jwt={accessToken}
     authContext="embed"
   >
-    {#if navigationEnabled}
-      <TopNavigationBarEmbed {instanceId} {activeResource} />
+    {#if showTopBar}
+      <div
+        class="flex items-center w-full pr-4 py-1 min-h-[2.5rem]"
+        class:border-b={!onProjectPage}
+      >
+        <TopNavigationBarEmbed
+          {instanceId}
+          {activeResource}
+          {navigationEnabled}
+        />
+      </div>
     {/if}
 
-    <slot />
+    <div class="flex h-full overflow-hidden">
+      <div class="flex-1 overflow-hidden">
+        <slot />
+      </div>
+      {#if $dashboardChat && activeResource?.kind === ResourceKind.Explore}
+        <ExploreChat />
+      {/if}
+    </div>
```

Comment:
This could also go in the Explore-specific layout, but this works given it's in the direction of where we're headed.